README.adoc

```diff
@@ -60,24 +60,27 @@ Serve the docs directory and navigate to it in a browser
 
 === Develop Tick
 
-REPL with nREPL server
+Jack in with cider or equivalent method in other IDEs, and start the cljs build with:
 
 ----
-make nrepl
+(figwheel-start!)
 ----
 
-connect to the printed port
 
-Once in, start cljs build with:
+Run all tests with:
 
 ----
-(figwheel-start!)
+make test-all
 ----
 
-Run all tests with:
+which will run clojure, clojurescript (node) and clojurescript (chrome) tests.
+Backend tests are currently running with https://github.com/lambdaisland/kaocha[Kaocha] while frontend tests are using figwheel-main and the cljs-test-runner.
+
+As long as you have you have started your repl with these aliases "-A:dev:test-clj", you can also run backend clojure tests in the repl with something like
 
 ----
-make test-all
+(require '[kaocha.repl :as kr])
```

Comment:
ah, perhaps user/test-all should be update to use this. as long as it does refresh first

Reply:
yeah ok I've done the change in user.clj to use kaocha also there.
It does mean that the user needs to start the repl with also the test profile though or the kaocha dependency won't be available, but it should be fine and it's documented as well.

Reply:
btw I also renamed it `test-clj` since it was maybe confusing to use `test-all` where `test-all` in the makefile means something quite different (runs for JS as well)